context/app/static/js/components/workspaces/WorkspaceLaunchStopButtons/WorkspaceLaunchStopButtons.tsx

```diff
@@ -24,12 +26,17 @@ function StopWorkspaceButton({
   handleStopWorkspace,
   button: ButtonComponent,
   isStoppingWorkspace,
+  checkMaxDatasets = false,
```

Comment:
If this is necessary can we just pass `hasMaxDatasets` to avoid calling `isWorkspaceAtDatasetLimit` here and in its parent?

Reply:
No longer an issue with `showStop` prop 🎉